### PR TITLE
Navbar Changes

### PR DESCRIPTION
**Pull request by AAkashyap1**

### CODE REVIEW

**Comment by nathangong** [comment]
the nav bar has 2 rows when the user is in a group, also the open icon looks a bit out of place:
image

**Comment by AAkashyap1** [comment]
> the nav bar has 2 rows when the user is in a group, also the open icon looks a bit out of place:
> image

Ok how does it look now?

**Comment by thecodingwizard** [comment]
I think the links are too narrow... is there a better navbar layout?

**Comment by nathangong** [comment]
maybe this one?
image
I can help work on it if needed since I did the current flyout menu

**Comment by AAkashyap1** [comment]
> maybe this one?
> image
> I can help work on it if needed since I did the current flyout menu

I think this would look good what do you think @thecodingwizard

**Comment by AAkashyap1** [comment]
> I think the links are too narrow... is there a better navbar layout?

Screen Shot 2021-09-03 at 6 08 48 PM
 or maybe this one?

**Comment by AAkashyap1** [comment]
Screen Shot 2021-09-03 at 6 10 16 PM
or maybe this one?

**Comment by nathangong** [comment]
it might get it bit tall with all those links

**Comment by AAkashyap1** [comment]
we could put ide and forum on the bottom thing

File: src/components/TopNavigationBar/TopNavigationBar.tsx [hunk 1]
```diff
@@ -138,123 +170,79 @@ export default function TopNavigationBar({
...
+                      <Transition
+                        as={Fragment}
+                        enter="transition ease-out duration-200"
+                        enterFrom="opacity-0 translate-y-1"
+                        enterTo="opacity-100 translate-y-0"
+                        leave="transition ease-in duration-150"
+                        leaveFrom="opacity-100 translate-y-0"
+                        leaveTo="opacity-0 translate-y-1"
+                      >
+                        <Popover.Panel static className="hidden md:block z-20 shadow-lg absolute left-1/2 transform -translate-x-1/2 mt-3 px-2 w-screen max-w-md sm:px-0 lg:max-w-3xl">
```
Comment (ryanchou-dev) [inline src/components/TopNavigationBar/TopNavigationBar.tsx:201]:
```suggestion
                        <Popover.Panel static className="hidden md:block z-20 shadow-lg absolute left-1/2 transform -translate-x-1/2 -mt-2 px-2 w-screen max-w-md sm:px-0 lg:max-w-3xl">
```
fix margin

**Review by ryanchou-dev — CHANGES_REQUESTED** [review]

**Review by thecodingwizard — APPROVED** [review]
code seems fine. didn't test UI, I'm on phone. merge if you think it's good :D

**Review by ryanchou-dev — APPROVED** [review]
lgtm!

**Comment by maggie-j-liu** [comment]
do you think the dark mode background should be a little lighter (compare with sections dropdown)?
image
image

**Comment by AAkashyap1** [comment]
Yeah it should be I'll change that

**Review by maggie-j-liu — APPROVED** [review]

**Comment by maggie-j-liu** [comment]
fix the merge conflict, then i can merge

**Pull request merged**